extensions/cornerstone/src/services/ViewportActionCornersService/ViewportActionCornersService.ts

```diff
@@ -35,6 +35,7 @@ class ViewportActionCornersService extends PubSubService {
     getState: getStateImplementation,
     setActionComponent: setActionComponentImplementation,
     setActionComponents: setActionComponentsImplementation,
+    clearActionComponents: clearActionComponentsImplementation,
```

Comment:
Since we use it like `viewportActionCornerService.` how about instead of `viewportActionCornerService.setActionComponent` we say `viewportActionCornerService.setComponent`? and similar for others